obs_screen_recognition_script.py

```diff
@@ -88,9 +88,13 @@ def main():
     with open("obs_screen_recognition_settings.json") as settings_file:
         application_settings = json.load(settings_file)
 
+    if not application_settings["screen_format"] in ["1440p","1080p"]:
+        println("Only 1440p or 1080p screen formats currently supported")
+        exit(1)
+
     print("Running with settings:", application_settings)
-    image_directory = application_settings["image_directory"]
-    mask_file = application_settings["mask_file"]
+    image_directory = "./hll_map_open_detection/" + application_settings["screen_format"]
+    mask_file = "./hll_map_open_detection/mask-" + application_settings["screen_format"] + ".png"
```

Comment:
Hardcoding this folder makes it difficult to use this repository for a different game (which I have had 1 or 2 people ask about). Should the folder name be a setting?

Reply:
Ok, I got an idea... have a look at my additional commits: if we also rename the script file to something like `obs_obfuscator` you can run the thing with:

```
python obs_obfuscator ./hll
```

BTW, you no longer need to be in the script folder to launch it